interactions/client.py

```diff
@@ -579,8 +586,34 @@ def decorator(coro: Coroutine) -> Any:
 
         return decorator
 
+    @staticmethod
+    def _find_command(commands: List, command: str) -> ApplicationCommand:
+        """
+        Iterates over commands and returns an ApplicationCommand if it matches the name from command
+
+        :ivar commands: The list of dicts to iterate through
+        :type commands: List[ApplicationCommand]
```

Comment:
Documentation mismatch with input args, same change as the typehint change above ^

Reply:
Additionally, please use either `` or :class:`` formatting on L592.